src/deepsparse/loggers/base_logger.py

```diff
@@ -25,40 +42,21 @@ class BaseLogger(ABC):
     for the loggers that support various monitoring services APIs.
     """
 
-    def __init__(self):
-        pass
-
-    @property
-    @abstractmethod
-    def identifier(self) -> str:
-        """
-        :return: The name of the monitoring service that the
-        BaseLogger uses to log the inference data
-        """
-        raise NotImplementedError()
-
-    @abstractmethod
-    def log_latency(
-        self, pipeline_name: str, inference_timing: "InferenceTimingSchema"  # noqa F821
-    ):
-        """
-        Logs the inference latency information to the appropriate monitoring service
-        :param pipeline_name: The name of the inference pipeline from which the
-            logger consumes the inference information to be monitored
-        :param inference_timing: pydantic model that holds the information about
-            the inference latency of a forward pass
-        """
-        raise NotImplementedError()
+    def __init__(self, config: Optional[dict] = None):
```

Comment:
let's remove config from the base class, if the base class isn't doing anything with it, then we shouldn't keep it at this level